### PR TITLE
added accordions components for services, added dummy data

### PR DESCRIPTION
**Pull request by GedasIncirauskas**
1. Created accordions components to fill services/about page.
2. Created dummy date to fill about page information.
3. Added styles for mobile as well.
4. Used `dangerouslySetInnerHTML` to make work with data more interesting.

Screenshot 2022-04-29 170428
Screenshot 2022-04-29 170505

### CODE REVIEW

File: src/components/accordion/Accordion.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  const toggle = id => {
+    if (clicked === id) {
+      return setClicked(null);
+    }
+    setClicked(id);
+  };
+
+  return (
+    <S.Container>
+      {data.length > 0 ? (
```
Comment (JuliaShavyrova) [inline src/components/accordion/Accordion.jsx]: expression can be simplified

File: src/components/accordionItem/AccordionItem.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  return (
+    <React.Fragment>
+      <S.Wrap onClick={() => toggle(data.id)}>
+        <S.Element>{data.id}</S.Element>
+        <h2>{data.title}</h2>
+        <S.BorderLine active={clicked === data.id} />
```
Comment (JuliaShavyrova) [inline src/components/accordionItem/AccordionItem.jsx]: `clicked === data.id` can be defined as variable and reused

File: src/components/accordionItem/AccordionItem.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  return (
+    <React.Fragment>
+      <S.Wrap onClick={() => toggle(data.id)}>
```
Comment (JuliaShavyrova) [inline src/components/accordionItem/AccordionItem.jsx]: `data` can be destructured

File: src/components/accordionItem/AccordionItem.jsx [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  return (
+    <React.Fragment>
```
Comment (JuliaShavyrova) [inline src/components/accordionItem/AccordionItem.jsx]: FYI: React.Fragment has shortened version, you can use both options

File: src/components/dropdown/Dropdown.jsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import * as S from './Dropdown.styles';
+
+const Dropdown = ({ data }) => {
+  return (
+    <S.Dropdown>
+      {data.map(el => (
```
Comment (JuliaShavyrova) [inline src/components/dropdown/Dropdown.jsx]: `el` can be destructured

File: src/pages/about/About.jsx [hunk 6]
```diff
@@ -1,5 +1,12 @@
+import { Accordion } from '../../components';
+
 const About = () => {
-  return <div>About!!!!</div>;
+  return (
+    <div>
+      <h1>Our company services:</h1>
+      <Accordion />
```
Comment (JuliaShavyrova) [inline src/pages/about/About.jsx]: please make shared component `Accordion` so you could reuse it everywhere with any data

File: src/utils/data.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+export const data = [
```
Comment (JuliaShavyrova) [inline src/utils/data.js]: please rename variable with more specific name

File: src/components/accordionItem/AccordionItem.jsx (hunk 4 above)
Comment (GedasIncirauskas) [reply src/components/accordionItem/AccordionItem.jsx]: Ok, I fixed to `<></>`. Looks nicer :)

File: src/utils/data.js (hunk 7 above)
Comment (GedasIncirauskas) [reply src/utils/data.js]: Ok, I fixed to `companyServiceData`

File: src/components/accordion/Accordion.jsx (hunk 1 above)
Comment (GedasIncirauskas) [reply src/components/accordion/Accordion.jsx]: Ok, thanks. I fixed like that: ` {companyServiceData.length ? (`

File: src/components/accordionItem/AccordionItem.jsx (hunk 2 above)
Comment (GedasIncirauskas) [reply src/components/accordionItem/AccordionItem.jsx]: Thanks, I created variable isClicked like that: `const isClicked = clicked === id`

File: src/components/accordionItem/AccordionItem.jsx (hunk 3 above)
Comment (GedasIncirauskas) [reply src/components/accordionItem/AccordionItem.jsx]: Ok, fixed to ` const { id, title, about } = data;`

File: src/pages/about/About.jsx (hunk 6 above)
Comment (GedasIncirauskas) [reply src/pages/about/About.jsx]: Ok, I made possibility to pass data via props ` <Accordion data={companyServiceData} />`

File: src/components/dropdown/Dropdown.jsx (hunk 5 above)
Comment (GedasIncirauskas) [reply src/components/dropdown/Dropdown.jsx]: Ok, I destructured like that: ` {data.map(({ id, description }) => (`

File: src/components/accordionItem/AccordionItem.jsx [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  const isClicked = clicked === id;
+
+  return (
+    <>
+      <S.Wrap onClick={() => toggle(id)}>
+        <S.Element>{id}</S.Element>
+        <h2>{title}</h2>
+        <S.BorderLine active={isClicked} />
+      </S.Wrap>
+      {isClicked && <Dropdown data={about} />}
```
Comment (JuliaShavyrova) [inline src/components/accordionItem/AccordionItem.jsx]: just a note - i'm a bit confused with name "dropdown". it means some content inside accordion here, but "dropdown" usually means some area with clickable options list

Comment (GedasIncirauskas) [reply src/components/accordionItem/AccordionItem.jsx]: I had in mind that is like drop down menu, but I can change to `Content` or `DropdownContent`. Which one would be better and cleaner?

Comment (GedasIncirauskas) [reply src/components/accordionItem/AccordionItem.jsx]: Changed to `AccordionItemContent` :)

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**